inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -32,7 +32,6 @@
 
 
 @hep.over('title_translations', '^242[10_][0_]')
```

Comment:
As discussed IRL, we have decided to keep them separate and have `titles`, `titles_en`, `titles_fr`.